### PR TITLE
Mark v1.19.0-rc3

### PR DESCRIPTION
**Pull request by SpicyLemon**
## Description

This PR marks `v1.19.0-rc3` in the changelog and release notes on the `release/v1.19.x` branch.

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit and integration [tests](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [x] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Review by Taztingo — APPROVED** [review]

**Review by nullpointer0x00 — APPROVED** [review]

**Comment by SpicyLemon** [comment]
I'm gonna hold off on this until after #2083 has merged, and been backported to `release/v1.19.0`. That way, the tag is on the commit from this PR, which makes it easier to draw a before/after line.

**Pull request merged**
